File: BeautySaloon/BeautySaloon.Angular/src/app/shared/models/schedule.ts
import { DayOff } from "./dayoff";
import { Master } from "./master";
import { WorkingDay } from "./working-day";

export interface Schedule {
    id: number | null;
    masterId: number | null;
    master: Master | null;
    workingDays: WorkingDay[] | null;
    dayOffs: DayOff[] | null;
  }

File: BeautySaloon/BeautySaloon.API/Entities/BeautySaloonContextEntities/Schedule.cs
namespace BeautySaloon.API.Entities.BeautySaloonContextEntities;

public class Schedule
{
    public int? Id { get; set; }
    public int? MasterId { get; set; }
    public Master? Master { get; set; }
    public List<WorkingDay>? WorkingDays { get; set; }
    public List<DayOff>? DayOffs { get; set; }
}